### PR TITLE
Add logo of some languages on the front page

### PR DESCRIPTION
**Pull request by Carreau**
see http://carreau.github.io/jupyter.github.io/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,6 +57,59 @@ <h2 class="text-center">
       </div>
     </div>
     <hr>
+      <div class="row logos">
+        <div class="col-md-3 text-center">
+            <img src='images/Julia_256x256.png' alt='Julia' />
```
Comment (minrk) [inline index.html]: Can you put `title` as well, so it shows the name on hover?

**Comment by minrk** [comment]
Why `lua.gif.1` instead of `lua.gif`?

**Comment by Carreau** [comment]
> Why lua.gif.1 instead of lua.gif?

Already a PNG, gif added by mistake.

File: index.html (hunk 1 above)
Comment (Carreau) [inline index.html]: Done

**Comment by zmughal** [comment]
@Carreau++! Thanks for adding these. :-) Someone pointed them out to me today.

**Pull request merged**